src/comments.ts

```diff
@@ -17,17 +17,17 @@ export const HadError = (user: string | undefined, error: string) => ({
 });
 
 export const CIFailed = (slug: string, user: string, ciUrl: string) => ({
-    tag: `gh-actions-complaint-${slug}`,
+    tag: `gh-actions-complaint-${slug.slice(0, 7)}`,
     status: `@${user} The CI build failed! Please [review the logs for more information](${ciUrl}).\r\n\r\nOnce you've pushed the fixes, the build will automatically re-run. Thanks!`
 });
 
 export const MergeConflicted = (slug: string, user: string) => ({
-    tag: `merge-complaint-${slug}`,
+    tag: `merge-complaint-${slug.slice(0, 7)}`,
     status: `@${user} Unfortunately, this pull request currently has a merge conflict 😥. Please update your PR branch to be up-to-date with respect to master. Have a nice day!`
 });
 
 export const ChangesRequest = (headOid: string, user: string) => ({
```

Comment:
I'd just change the argument here since it was confusing to call it `headOid` when its the abbreviated one.

Reply:
Done :heavy_check_mark: